Scala/Scala.sublime-syntax

```diff
@@ -130,6 +131,48 @@ contexts:
     - match: '{{typeprefix}}'
       push: single-type-expression
 
+  annotation:
+    - match: '@specialized'
```

Comment:
>  other annotations *can* take type parameters in this manner

I'm not really familiar with this kind of annotations. I mainly use annotations with string/int parameters. I haven't found documentation on annotations having types inside "()".
If you have pointers or code sample, I'll be happy to have a look at them.

My plan is to support  `@test[A, B]`, `@test(1, "foo")` and `@test(a, b)`
But I'm not sure how to handles `@test(Int, Foo)` because it's not really different from the previous.